Source/WebCore/rendering/FloatingObjects.h

```diff
@@ -103,6 +104,9 @@ class FloatingObject {
     LayoutSize translationOffsetToAncestor() const;
 
 private:
+    friend FloatingObjects;
+    // ideally friend FloatingObjects::shiftFloatsBy(LayoutUnit); but that won't compile...
```

Comment:
I doubt we want to leave this comment here. If we really do then please start with a capital letter and end with a (single) period.

Reply:
Removed, that was mainly for review actually...